ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6620,3 +6620,27 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+
+SEWTM:
```

Comment:
This type already exists on line 6509. Please sync your fork with the latest master branch.